mod/state-transition/pkg/core/state_processor_payload.go

```diff
@@ -46,6 +48,14 @@ func (sp *StateProcessor[
 		g, gCtx = errgroup.WithContext(context.Background())
 	)
 
+	sp.logger.Info("processExecutionPayload",
+		fmt.Sprintf("consensus height %d", ctx.GetConsensusBlockHeight()),
+		fmt.Sprintf("payload height %d", payload.GetNumber()),
+		fmt.Sprintf("payload timestamp %d", payload.GetTimestamp()),
+		fmt.Sprintf("bound timestamp %d", payload.GetTimestamp()),
+		fmt.Sprintf("skip verification %t", ctx.GetSkipPayloadVerification()),
+	)
```

Comment:
without an explicitly formatting via fmt.Sprintf, logging seems to show the hexadecimal values of these quantities rather than their decimal representation